js/runner_test.go

```diff
@@ -534,6 +535,56 @@ func TestVURunInterrupt(t *testing.T) {
 	}
 }
 
+func TestVURunInterruptDoesntPanic(t *testing.T) {
+	//TODO: figure out why interrupt sometimes fails... data race in goja?
```

Comment:
I think the issue would disappear after we update goja to include this fix: https://github.com/dop251/goja/commit/f95411d186f2183429261010a860c604371ac6a7

Reply:
i think we even have an issue for this ... maybe we should upgrade goja ... soon :tm: